include/zpios-ctl.h

```diff
@@ -146,10 +146,6 @@ zpios_timespec_normalize(zpios_timespec_t *ts, uint32_t sec, uint32_t nsec)
 		nsec -= NSEC_PER_SEC;
 		sec++;
 	}
-	while (nsec < 0) {
-		nsec += NSEC_PER_SEC;
-		sec--;
-	}
```

Comment:
We want to preserve this logic to ensure the value get's normalized properly but in order for that to work these will need to be treated as signed values.